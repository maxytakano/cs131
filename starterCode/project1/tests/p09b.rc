function : float foo(float x) { return 0.0 }
function : float foo(float x, float y, float z) { return 0.0  }
function : int main() {
 foo(1); // error9_Illegal
 return 0;
}