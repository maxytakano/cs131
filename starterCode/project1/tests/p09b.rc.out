Error, "/tmp/cs131_testrunner_yyTaux/file.rc", line 1: 
  syntax error near "}".
Compile: failure.
